code/game/gamemodes/cult/cultify/obj.dm

```diff
@@ -12,22 +14,33 @@
 
 /obj/item/device/flashlight/lamp/cultify()
 	new /obj/structure/cult/pylon(get_turf(src))
-	..()
+	..(FALSE)
+	return "As you cast \the [src] into the forge, a great pylon grows beneath your feet, achieving its form in seconds."
 
 /obj/item/stack/material/wood/cultify()
 	return
 
 /obj/item/book/cultify()
-	new /obj/item/book/tome(get_turf(src))
-	..()
+	var/obj/item/book/tome/T = new /obj/item/book/tome(get_turf(src))
+	..(FALSE)
```

Comment:
At this point don't add a default value to the cultify proc if you're always overriding it